soc/arm/st_stm32/stm32g0/power.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics.
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <pm/pm.h>
+#include <soc.h>
+#include <init.h>
+
+#include <stm32g0xx_ll_utils.h>
+#include <stm32g0xx_ll_bus.h>
+#include <stm32g0xx_ll_cortex.h>
+#include <stm32g0xx_ll_pwr.h>
+#include <stm32g0xx_ll_rcc.h>
```

Comment:
Is this one still required ?

Reply:
Seems like not required, I've removed it.
Thanks for the inputs @erwango @str4t0m @FRASTM